documentation/homework/homeworkweek1/App.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const ListItem = props => {
+  const { description, deadline } = props.listInfo;
+  return (
+    <li>
+      {' '}
+      {description},{deadline}
+    </li>
+  );
+};
+
+const StaticList = props => {
+  return (
+    <div className="static-list">
+      <p>Static List</p>
+
+      <ListItem listInfo={{ description: ' Get out of bed', deadline: 'Wed Sep 13 2017' }} />
+      <ListItem listInfo={{ description: 'Brush teeth', deadline: 'Thu Sep 14 2017' }} />
+      <ListItem listInfo={{ description: ' Eat breakfast', deadline: 'Fri Sep 15 2017' }} />
+    </div>
+  );
+};
+
+const DynamicList = props => {
+  const { DynamicListInfo } = props;
+  return (
+    <div className="dynamic-list">
+      <p>Dynamic List</p>
+      {DynamicListInfo.map((item, index) => (
+        <li key={index} className={item.done ? 'done' : 'not-done'}>
+          {item.description},{item.deadline}
```

Comment:
Two notes are here:
1. It would be possible to re-use `ListItem` instead as the `StaticList` Component since they have the same result: 
```javascript
       <ListItem key={index} listInfo={item} />
```
and it would be nice if you have got the `className` there too 😊

2. It's super clever to solve the problem with the most simplest way, which I like to do, and it's not mandatory for this small application but in a production you will have an `id` for each item just as this one.
the index may cause bugs it's possible to run out of them by using `id` instead.